parquet-common/src/main/java/org/apache/parquet/ParquetRuntimeException.java

```diff
@@ -43,4 +46,18 @@ public ParquetRuntimeException(Throwable cause) {
     super(cause);
   }
 
+  /**
+   * Call the #close() method on a {@see Closable}, wrapping any IOException
+   * in a runtime exception.
+   *
+   * @param closeable - resource to close
+   */
+  public static void closeQuietly(Closeable closeable) {
```

Comment:
Wasn't quite sure about a better place to put it, is there somewhere for utilities like this?